cgi-bin/LJ/Protocol.pm

```diff
@@ -2157,8 +2166,8 @@ sub editevent
             $u->do( 'INSERT INTO logslugs (journalid, jitemid, slug) VALUES (?, ?, ?)',
                     undef, $ownerid, $itemid, $slug );
             if ( $u->err ) {
-                $res->{message} ||= 'Sorry, it looks like that slug has already been used. ' .
-                    'Your entry has been updated, but you can still edit it again to add a unique slug.';
+                $add_message->( 'Sorry, it looks like that slug has already been used. ' .
+                    'Your entry has been updated, but you can still edit it again to add a unique slug.' );
```

Comment:
If you're using this add_message subroutine to potentially return multiple errors from one edit, that would be another argument in favor of breaking up the messages onto multiple lines.